tokio/src/net/tcp/stream.rs

```diff
@@ -154,7 +155,7 @@ impl TcpStream {
 
     pub(crate) fn new(connected: mio::net::TcpStream) -> io::Result<TcpStream> {
         let io = PollEvented::new(connected)?;
-        Ok(TcpStream { io })
+        Ok(new_instrumented_resource!(Network, TcpStream { io }))
```

Comment:
the `Network` here becomes the span's `kind` field, correct? so, it will become a string "Network". the instrumented task spans _also_ have a `kind` field, but those are currently all lowercase (such as `kind="task"` and `kind="blocking"` etc). i don't have a particularly strong preference for whether the `kind` fields are all uppercase or all lowercase, but i do think they ought to be consistent between tasks and resources. can we either change these to be lowercase, or change the task spans' `kind` field to be capitalized?